ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -479,7 +479,7 @@ QtObject {
             const userCanJoin = userCanJoin(result.communityData.communityId)
             // TODO find what to do when you can't join
             if (userCanJoin) {
-                requestToJoinCommunityWithAuthentication(userProfileInst.preferredName) // FIXME what addresses to share?
+                requestToJoinCommunityWithAuthentication(result.communityId, userProfileInst.preferredName) // FIXME what addresses to share?
```

Comment:
This looks like a bad merge; it should be:

```qml
requestToJoinCommunityWithAuthentication(result.communityData.communityId, userProfileInst.preferredName)
```
`result.communityId` is something that existed before this PR

Reply:
done